reactive-mongo/src/main/scala/acolyte/reactivemongo/Akka.scala

```diff
@@ -49,17 +49,17 @@ private[reactivemongo] class Actor(
 
   import reactivemongo.core.nodeset.{ Authenticate, ChannelFactory, Connection }
 
-  lazy val initialAuthenticates = Seq.empty[Authenticate]
+  override lazy val initialAuthenticates = Seq.empty[Authenticate]
```

Comment:
`initialAuthenticates` is an abstract `def` in `MongoDBSystem` so `override` is meaningless